Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -133,7 +133,7 @@ var DatePickerIOS = React.createClass({
           mode={props.mode}
           minuteInterval={props.minuteInterval}
           timeZoneOffsetInMinutes={props.timeZoneOffsetInMinutes}
-          onChange={this._onChange}
+          onDateChange={this._onChange}
```

Comment:
I'm pretty sure this change is also wrong - there is no onDateChange event exported by the native component.